buildstockbatch/hpc.py

```diff
@@ -563,7 +563,7 @@ def queue_jobs(self, array_ids=None, hipri=False):
         job_id = m.group(1)
         return [job_id]
 
-    def queue_post_processing(self, after_jobids=[], upload_only=False, hipri=False):
+    def queue_post_processing(self, after_jobids=[], upload_only=False, hipri=False, continue_upload=False):
```

Comment:
Since our workflow calls queue_post_processing no more than once, it shouldn't be an issue. Regardless, agreed that it's a good practice to avoid mutable defaults.